tests/unit/test_federated_s3.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 - 2024 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the FederatedS3Fixture and related items."""
+
+import pytest
+
+from hexkit.providers.s3 import S3Config
+from hexkit.providers.s3.testutils import (  # noqa: F401
+    FederatedS3Fixture,
+    federated_s3_fixture,
+    s3_multi_container_fixture,
+)
+from hexkit.providers.s3.testutils._fixtures import S3MultiContainerFixture
+
+pytestmark = pytest.mark.asyncio()
+
+PRIMARY_STORAGE_ALIAS = "primary"
+SECONDARY_STORAGE_ALIAS = "secondary"
+STORAGE_ALIASES = [PRIMARY_STORAGE_ALIAS, SECONDARY_STORAGE_ALIAS]
+
+
+@pytest.fixture(scope="session")
+def storage_aliases():
+    """Return the storage aliases for the federated S3 fixture."""
+    return STORAGE_ALIASES
+
+
+async def test_get_configs_by_alias(federated_s3: FederatedS3Fixture):
+    """Test that `get_configs_by_alias` returns the configs of each storage by name."""
+    configs = federated_s3.get_configs_by_alias()
+    assert configs.keys() == set(STORAGE_ALIASES)
```

Comment:
```suggestion
    assert set(configs) == set(STORAGE_ALIASES)
```

Since it's not obvious that you can compare a view object with a set.